### PR TITLE
refactor: clean and simplify venom codebase

### PR DESCRIPTION
**Pull request by fsamin**
Signed-off-by: francois  samin <francois.samin@corp.ovh.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-venom#270.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘  
  * Lib Unit Tests ✔

File: cli/venom/run/cmd.go [hunk 1]
```diff
@@ -37,12 +37,11 @@ import (
...
-	variables       []string
-	exclude         []string
-	format          string
-	varFiles        []string
-	withEnv         bool
+	path      []string
+	variables []string
+	format    string
+	varFiles  []string
+	//withEnv         bool
```
Comment (yesnault) [inline cli/venom/run/cmd.go:44]: to delete

File: cli/venom/run/cmd.go [hunk 2]
```diff
@@ -56,9 +55,7 @@ var (
 func init() {
 	Cmd.Flags().StringSliceVarP(&variables, "var", "", []string{""}, "--var cds='cds -f config.json' --var cds2='cds -f config.json'")
 	Cmd.Flags().StringSliceVarP(&varFiles, "var-from-file", "", []string{""}, "--var-from-file filename.yaml --var-from-file filename2.yaml: hcl|json|yaml, must contains map[string]string'")
```
Comment (yesnault) [inline cli/venom/run/cmd.go:57]:
```suggestion
	Cmd.Flags().StringSliceVarP(&varFiles, "var-from-file", "", []string{""}, "--var-from-file filename.yaml --var-from-file filename2.yml: json|yml, must contains map[string]string'")
```

**Review by yesnault — CHANGES_REQUESTED** [review]

File: cli/venom/run/cmd.go (hunk 1 above)
Comment (yesnault) [inline cli/venom/run/cmd.go:44]:
```suggestion
```

File: cli/venom/run/cmd.go [hunk 3]
```diff
@@ -119,87 +116,112 @@ Notice that variables initialized with -var-from-file argument can be overrided
...
-				log.Fatal("unsupported varFile format")
-			}
+			fi, err := os.Open(f)
 			if err != nil {
-				log.Fatal(err)
-			}
-
-			for key, value := range varFileMap {
-				mapvars[key] = value
+				return fmt.Errorf("unable to open var-file %s: %v", f, err)
```
Comment (yesnault) [inline cli/venom/run/cmd.go:142]:
```suggestion
				return fmt.Errorf("unable to open var-from-file %s: %v", f, err)
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-venom#272.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘  
  * Lib Unit Tests ✔

**Review by yesnault — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-venom#273.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘  
  * Lib Unit Tests ✔

**Comment by ovh-cds** [comment]
CDS Report build-venom#275.0 ✘  
* Build
  * Build ✘

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-venom#276.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘  
  * Lib Unit Tests ✔

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-venom#277.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘  
  * Lib Unit Tests ✔

**Comment by ovh-cds** [comment]
CDS Report build-venom#278.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘  
  * Lib Unit Tests ✔

**Pull request merged**